unitTest/@testFEMComputationalFluidDynamicsAnalysis/testFEM4NavierStokesSteadyState2DReferencePaper.m

```diff
@@ -0,0 +1,146 @@
+function testFEM4NavierStokesSteadyState2DReferencePaper(testCase)
```

Comment:
Change the name of the function testFEM4NavierStokesSteadyState2DReferencePaper to testFEM4NavierStokesSteadyStateFlowAroundCylinder2D